### DISCUSSION TITLE
Configure date format in CTRL-R

### DISCUSSION THREAD

**Discussion by Scartiloffista** [PatrickF1/fzf.fish#261] [Archive]
Right now the timestamp is fixed as MM-DD, and doesn't seem configurable through `fzf_history_opts`. It would be cool to be able to configure that!

**Comment by PatrickF1** [comment]
Hi, the timestamp is showing up for you as MM-DD HH:MM:SS right?

Yeah it would be nice, but then it'd break the logic that removes the timestamp from the lines selected before adding them to the command line. Not saying it's not doable but it's not a trivial thing to do. I'll keep this on the ideas list!

**Reply by Scartiloffista** [reply]
oh yes, i've forgot about HH etc :D thanks, anyway!

**Reply by PatrickF1** [reply]
This was actually done in https://github.com/PatrickF1/fzf.fish/pull/299

**Comment by Scartiloffista** [comment]
well, super, thanks!
